src/auto-evo/RunResults.cs

```diff
@@ -282,6 +290,13 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                         OutputPopulationForPatch(entry.Species, patchPopulation.Key, adjustedPopulation);
                 }
 
+                if (GetGlobalPopulation(entry.Species) == 0)
+                {
+                    builder.Append("  ");
+                    builder.Append(" went extinct from the world");
```

Comment:
I think `planet` should be used instead of world.

Reply:
@Straywind This still hasn't been changed